tests/unit/test_graph_runner.py

```diff
@@ -53,3 +54,46 @@ def test_traverse_and_execute_analyze(stub_graph_set):
     assert iso_relation.target_loaded is False
     assert iso_relation.sample_size == 100
     assert iso_relation.population_size == 1000
+
+
+def test_traverse_and_execute_custom_max_rows_pass(stub_graph_set):
+    """
+    Tests if the values stated in config are correctly passed to check_count_and_query() method
+    """
+
+    source_adapter,target_adapter=[mock.MagicMock() for _ in range(2)]
+    source_adapter.predicate_constraint_statement.return_value=str()
+    source_adapter.upstream_constraint_statement.return_value=str()
+    source_adapter.union_constraint_statement.return_value=str()
+    source_adapter.sample_statement_from_relation.return_value=str()
+    runner=GraphSetRunner()
+    runner.barf=False
+    graph_set,vals=stub_graph_set
+    source_adapter.scalar_query.return_value=1000
+    source_adapter.check_count_and_query.return_value=pd.DataFrame([dict(population_size=1000,sample_size=100)])
+    dag=copy.deepcopy(graph_set[-1]) # last graph in the set is the dag
+    
+    # test if defaults are passed
+    for rel in dag.nodes:
+        rel.unsampled=False
+        rel.include_outliers=False
+        rel.sampling=DefaultSampling()
+
+    dag_executable = GraphExecutable(dag, source_adapter, target_adapter, True)
```

Comment:
it looks like in both cases the analyse attribute of the GraphExecutable is defined as True;
Meantime, there is uncovered call to 'check_count_and_query' when the analyse attribute eq to False
for more details please check the `_traverse_and_execute` function of the GraphSetRunner class

Reply:
Done